rm-datasource/src/main/java/io/seata/rm/datasource/undo/oracle/OracleUndoLogManager.java

```diff
@@ -99,4 +100,8 @@ private void insertUndoLog(String xid, long branchID, String rollbackCtx, byte[]
         }
     }
 
+    @Override
+    protected String getCheckUndoLogTableExistSql() {
+        return CHECK_UNDO_LOG_TABLE_EXIST_SQL;
+    }
```

Comment:
How about using the following SQL statement instead of throwing an exception to determine whether a table exists?
```
select count(*) from user_tables where table_name =upper('undo_log_table_name')
```

Reply:
select 1 from {targetTable} is the fastest way to figure out whether target table exists or not. Since we don't care what it "looks like", no need to query user_tables.